tests/test_convert.py

```diff
@@ -1,11 +1,28 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
-import os
+
 import subprocess
 from pathlib import Path
+from click.testing import CliRunner
 
 import pytest
 
+from dist2src.cli import cli
+from packit.cli.packit_base import packit_base
+from packit.utils import cwd
+
+
+def run_dist2src(*args, working_dir=None, **kwargs):
+    working_dir = working_dir or Path()
+    with cwd(working_dir):
+        cli_runner = CliRunner()
+        cli_runner.invoke(cli, *args, catch_exceptions=False, **kwargs)
+
+
+def run_packit(*args, **kwargs):
+    cli_runner = CliRunner()
+    cli_runner.invoke(packit_base, *args, catch_exceptions=False, **kwargs)
```

Comment:
I actually don't like this because the click runner does not support ipdb debugger; luckily I found a solution how to do it with plain pdb which is good enough for me

Reply:
Aha. Sorry... PyCharm's debuger plays well with that. I am wondering who needs to fix that -- ipdb x click (I know they're doing a bit of magic here).

The reason why I really like this way is that you can test CLI but still receives regular exceptions and that (my..;) debugger still works.

Reply:
looks like it's time for me to start using pycharm debugger then :)